### ISSUE TITLE
many object detecrtion

### ISSUE THREAD

**Issue by itkt** [Ghustwb/MobileNet-SSD-TensorRT#13]
Hi, Thanks for work. Very fast work. But after starting the program a lot of objects detection (demo data):
`allocate output
0 , 0 , 0.126316 , 0.126316
0.00823669 , 0 , 0.149658 , 0.167737
0.0315789 , 0 , 0.231579 , 0.126316
0.1135 , 0 , 0.254921 , 0.167737
0.136842 , 0 , 0.336842 , 0.126316
0.218763 , 0 , 0.360184 , 0.167737
0.242105 , 0 , 0.442105 , 0.126316
0.324026 , 0 , 0.465447 , 0.167737
0.347368 , 0 , 0.547368 , 0.126316
0.429289 , 0 , 0.570711 , 0.167737
0.452632 , 0 , 0.652632 , 0.126316
0.534552 , 0 , 0.675974 , 0.167737
0.557895 , 0 , 0.757895 , 0.126316
0.639816 , 0 , 0.781237 , 0.167737
0.663158 , 0 , 0.863158 , 0.126316
0.745079 , 0 , 0.8865 , 0.167737
0.768421 , 0 , 0.968421 , 0.126316
0.850342 , 0 , 0.991763 , 0.167737
0.873684 , 0 , 1 , 0.126316
0 , 0.00823669 , 0.167737 , 0.149658
0.0427892 , 0.00823669 , 0.325632 , 0.149658
0.200684 , 0.00823669 , 0.483527 , 0.149658
0.358579 , 0.00823669 , 0.641421 , 0.149658
0.516473 , 0.00823669 , 0.799316 , 0.149658
0 , 0 , 0.126316 , 0.126316
0.00823669 , 0 , 0.149658 , 0.167737
0.0315789 , 0 , 0.231579 , 0.126316
0.1135 , 0 , 0.254921 , 0.167737
0.136842 , 0 , 0.336842 , 0.126316
0.218763 , 0 , 0.360184 , 0.167737
0.242105 , 0 , 0.442105 , 0.126316
0.324026 , 0 , 0.465447 , 0.167737
0.347368 , 0 , 0.547368 , 0.126316
0.429289 , 0 , 0.570711 , 0.167737
0.452632 , 0 , 0.652632 , 0.126316
0.534552 , 0 , 0.675974 , 0.167737
0.557895 , 0 , 0.757895 , 0.126316
0.639816 , 0 , 0.781237 , 0.167737
0.663158 , 0 , 0.863158 , 0.126316
0.745079 , 0 , 0.8865 , 0.167737
0.768421 , 0 , 0.968421 , 0.126316
0.850342 , 0 , 0.991763 , 0.167737
0.873684 , 0 , 1 , 0.126316
0 , 0.00823669 , 0.167737 , 0.149658
0.0427892 , 0.00823669 , 0.325632 , 0.149658
0.200684 , 0.00823669 , 0.483527 , 0.149658
0.358579 , 0.00823669 , 0.641421 , 0.149658
0.516473 , 0.00823669 , 0.799316 , 0.149658
0 , 0 , 0.126316 , 0.126316
0.00823669 , 0 , 0.149658 , 0.167737
0.0315789 , 0 , 0.231579 , 0.126316
0.1135 , 0 , 0.254921 , 0.167737
0.136842 , 0 , 0.336842 , 0.126316
0.218763 , 0 , 0.360184 , 0.167737
0.242105 , 0 , 0.442105 , 0.126316
0.324026 , 0 , 0.465447 , 0.167737
0.347368 , 0 , 0.547368 , 0.126316
0.429289 , 0 , 0.570711 , 0.167737
0.452632 , 0 , 0.652632 , 0.126316
0.534552 , 0 , 0.675974 , 0.167737
0.557895 , 0 , 0.757895 , 0.126316
0.639816 , 0 , 0.781237 , 0.167737
0.663158 , 0 , 0.863158 , 0.126316
0.745079 , 0 , 0.8865 , 0.167737
0.768421 , 0 , 0.968421 , 0.126316
0.850342 , 0 , 0.991763 , 0.167737
0.873684 , 0 , 1 , 0.126316
0 , 0.00823669 , 0.167737 , 0.149658
0.0427892 , 0.00823669 , 0.325632 , 0.149658
0.200684 , 0.00823669 , 0.483527 , 0.149658
0.358579 , 0.00823669 , 0.641421 , 0.149658
0.516473 , 0.00823669 , 0.799316 , 0.149658
0 , 0 , 0.126316 , 0.126316
0.00823669 , 0 , 0.149658 , 0.167737
0.0315789 , 0 , 0.231579 , 0.126316
0.1135 , 0 , 0.254921 , 0.167737
0.136842 , 0 , 0.336842 , 0.126316
0.218763 , 0 , 0.360184 , 0.167737
0.242105 , 0 , 0.442105 , 0.126316
0.324026 , 0 , 0.465447 , 0.167737
0.347368 , 0 , 0.547368 , 0.126316
0.429289 , 0 , 0.570711 , 0.167737
0.452632 , 0 , 0.652632 , 0.126316
0.534552 , 0 , 0.675974 , 0.167737
0.557895 , 0 , 0.757895 , 0.126316
0.639816 , 0 , 0.781237 , 0.167737
0.663158 , 0 , 0.863158 , 0.126316
0.745079 , 0 , 0.8865 , 0.167737
0.768421 , 0 , 0.968421 , 0.126316
0.850342 , 0 , 0.991763 , 0.167737
0.873684 , 0 , 1 , 0.126316
0 , 0.00823669 , 0.167737 , 0.149658
0.0427892 , 0.00823669 , 0.325632 , 0.149658
0.200684 , 0.00823669 , 0.483527 , 0.149658
0.358579 , 0.00823669 , 0.641421 , 0.149658
0.516473 , 0.00823669 , 0.799316 , 0.149658`

**Comment by PiyalGeorge** [comment]
@itkt , What all changes did you do in pluginImplement.cpp and pluginImplement.h?

**Comment by itkt** [comment]
I do not change this file

**Comment by PiyalGeorge** [comment]
@itkt , thanks for the reply. did you try with a custom caffemodel or the same model in this repo?

**Comment by itkt** [comment]
I try with model in this repo. And now I try with a custom model (trained Mobilenet ssd), and have a problem `CaffeParser: ERROR: Attempting to access NULL weights
Weights for layer conv1/dw doesn't exist` and this message for all layer

**Comment by PiyalGeorge** [comment]
I trained mobilenet-ssd model in caffe following [this](https://github.com/chuanqi305/MobileNet-SSD) . This guy @Ghustwb, has mentioned in the ReadMe that he has trained his model(see his model in model folder in the repo) with 5 classes. So i also trained my mobilenet-ssd model with 5 classes(including background). then i used that 5 class model and tried that worked successfully. Try that. But currently i'm trying to do the same for 7 classes. the Ghustwb has specified some method [here](https://github.com/Ghustwb/MobileNet-SSD-TensorRT/issues/9) . But i think its **not** working.

**Comment by itkt** [comment]
Yes, We also trained model following [tutorial](https://github.com/chuanqi305/MobileNet-SSD). But we need only 2 class. After training we loaded weights into this project. Then start ./mobilenet have error. Can you help me start the application with model trained to 2 classes

**Comment by PiyalGeorge** [comment]
@itkt believe me my friend, I've also been trying to get the same thing. If you see the issue list raised in this repo, you can see my issue for this. And Ghustwb has replied. But couldn't get to the exact result and failed to fix the issue. If I'm able to fix the  issue, I'll surely tell you. If you get to fix it first, I would be thankful if you share the details.
